extension-push/src/push_utils.cpp

```diff
@@ -63,11 +68,17 @@ static void HandleRegistrationResult(const dmPush::Command* cmd)
     (void)ret;
 
     dmScript::TeardownCallback(cmd->m_Callback);
+    dmScript::DestroyCallback(cmd->m_Callback);
 }
 
 
 static void HandlePushMessageResult(const dmPush::Command* cmd, bool local)
 {
+    if (!dmScript::IsCallbackValid(cmd->m_Callback))
```

Comment:
Not related to this issue, but same rule applies: Early exit if callback isn't valid